src/index.ts

```diff
@@ -1,3 +1,10 @@
+export type {
+  UUIDTypes,
+  Version1Options,
+  Version4Options,
+  Version6Options,
+  Version7Options,
+} from './_types.js';
```

Comment:
```suggestion
export type * from './_types.js';
```

This appears to work and will be a bit more future proof as types get added / removed. (please test for your use case, however)

Note:  "_types" name implies these are _private_ types, which is no longer the case.   Please rename src/_types.ts to src/types.ts, and updated references to it accordingly.

Reply:
@broofa This change breaks compatibility with older Typescript versions, as export type * is only allowed since Typescript v5. See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#support-for-export-type-.

Maybe revert back to the explicit export?